src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -42,13 +42,13 @@
 import java.util.TreeSet;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 import net.sf.jabref.event.source.EntryEventSource;
 import net.sf.jabref.model.entry.BibEntry;
 import net.sf.jabref.model.entry.BibtexString;
 import net.sf.jabref.model.entry.EntryUtil;
 import net.sf.jabref.model.entry.FieldName;
-import net.sf.jabref.model.entry.InternalBibtexFields;
```

Comment:
I wrote the isInternalField method in that  class and not used it from InternalBibtexFields, because I thought that ``isInternalField(field)`` seems more readable then ``InternalBibTexField.isInternalField(field)``.